feature-utils/silly-i18n/.eslintrc.cjs

```diff
@@ -9,7 +9,7 @@ module.exports = {
             files: ["*.js"],
             parserOptions: {
                 sourceType: "module",
-                ecmaVersion: 2020
+                ecmaVersion: 2021
```

Comment:
Is there some specific thing you're using that needs this change?

Reply:
Yes, the `||=` operator.

Reply:
Well... if linting fails we will generally fix whatever it reveals, not change linting configuration; even less so in a specific module. We might eventually move up to 2021 for the whole repository, but I'm not sure this is the time.

Reply:
It's not a liniting error that I'm overriding. It's just a new syntax feature (i.e. a new operator) that would come in handy and is supported in all browsers we support as far as I know,  please correct me if wrong. However, if parsing the code as ES2020, ESlint is bailing with a syntax error.

Reply:
Again, not the time (this PR) or location (this specific module) for changing linting configuration. We might eventually move to ES2021, but for the time being it's best if we stick to using the operators that are available in the version we use everywhere. I would appreciate if you opened an issue requesting a general change in configuration, though; I really think we shouldn't stay forever in a specific version.

Reply:
To be honest, at this point there are practically no versions of JavaScript anymore. Browsers adapt new features progressively, and everyone picks them up as they are supported by the browsers they care about. Also mind that this setting is only about the parser, i.e. only affects JavaScript features that are part of the syntax of the language. So I would just bump it up as needed, while making sure that the new JavaScript feature introduced are supported in all browsers that we support. Rather than waiting until all features in a particular snapshot of the ES standard has been implemented by all browsers we support.

Reply:
I see what you mean. Still, bumping up standards is probably not in the scope of this PR.

Reply:
Agreed, I'd also prefer not to include this. Though if we're reasonably sure that the syntax doesn't end up in the generated code (thanks to sucrase, I think it shouldn't), it doesn't kill me. But it's definitely weird to have different parts of the code base being able to use different ES features.

Reply:
Fair enough. I got rid of the `||=` operator, though I don't quite like the resulting boilerplate: 80153a7

I submitted #878 for updating the ESLint configuration to use ES2021.